home/export_assessments.py

```diff
@@ -0,0 +1,258 @@
+        _set_xlsx_styles(workbook, worksheet)  # type: ignore
+        workbook.save(response)
+
+    def write_csv(self, response: HttpResponse) -> None:
+        csv_response = csv.writer(response)
+        csv_response.writerow(ExportRow.headings())
+
+        # Write data rows
+        for row in self.rows:
+            row_values = [
+                row["title"],
+                row["page_id"],
+                row["tags"],
+                row["slug"],
+                row["locale"],
+                row["high_result_page"],
+                row["high_inflection"],
+                row["medium_result_page"],
+                row["medium_inflection"],
+                row["low_result_page"],
+                row["low_inflection"],
+                row["generic_error"],
+                row["question_count"],
+                row["questions"],
+                row["error"],
+                row["answers"],
+                row["score"],
+            ]
+            csv_response.writerow(row_values)
+
+
+def _set_xlsx_styles(wb: Workbook, sheet: Worksheet) -> None:
+    """Sets the style for the workbook adding any formatting that will make the sheet more aesthetically pleasing"""
+    # Adjustment is because the size in openxlsx and google sheets are not equivalent
+    adjustment = 7
+    # Padding
+    sheet.insert_cols(1)
+
+    # Set columns based on best size
+
+    column_widths_in_pts = {
+        "title": 110,
+        "page_id": 110,
+        "tags": 110,
+        "slug": 110,
+        "locale": 118,
+        "high_result_page": 110,
+        "high_inflection": 110,
+        "medium_result_page": 110,
+        "medium_inflection": 110,
+        "low_result_page": 118,
+        "low_inflection": 110,
+        "generic_error": 300,
+        "question_count": 110,
+        "questions": 370,
+        "error": 400,
+        "answer": 110,
+        "score": 110,
+    }
+
+    for index, column_width in enumerate(column_widths_in_pts.values(), 2):
+        sheet.column_dimensions[get_column_letter(index)].width = ceil(
+            column_width / adjustment
+        )
+
+    # Freeze heading row and side panel, 1 added because it freezes before the column
+    panel_column = get_column_letter(5)
+    sheet.freeze_panes = sheet[f"{panel_column}2"]
+
+    # Colours
+    blue = Color(rgb="0099CCFF")
+
+    # Boarders
+    left_border = Border(left=Side(border_style="thin", color="FF000000"))
+
+    # Fills
+    blue_fill = PatternFill(patternType="solid", fgColor=blue)
+
+    # Named Styles
+    header_style = NamedStyle(name="header_style")
+    menu_style = NamedStyle(name="menu_style")
+
+    # Set attributes to styles
+    header_style.font = Font(bold=True, size=10)
+    menu_style.fill = blue_fill
+    menu_style.font = Font(bold=True, size=10)
+
+    # Add named styles to wb
+    wb.add_named_style(header_style)
+    wb.add_named_style(menu_style)
+
+    # column widths
+
+    # Set menu style for any "Menu" row
```

Comment:
I don't think we have any `Menu` rows in the assessment exports, so we can get rid of this styling